src/server/api/match-maker.ts

```diff
@@ -1,4 +1,5 @@
 import { EventEmitter } from "events";
+import schedule from "node-schedule";
```

Comment:
Should we use: https://www.npmjs.com/package/cron instead of "node-schedule", "cron" is more actively maintained, more downloads and less open issues.